framework/src/modules/chain/transport/broadcaster.js

```diff
@@ -28,41 +28,28 @@ const jobsQueue = require('../utils/jobs_queue');
  * @requires lodash
  * @requires utils/jobs_queue
  * @param {Object} broadcasts
- * @param {boolean} force
- * @param {Transaction} transaction - Transaction instance
  * @param {Object} logger
  * @todo Add description for the params
  */
 class Broadcaster {
-	constructor(broadcasts, transactionPool, logger, channel, storage) {
+	constructor({ broadcasts, transactionPool, logger, channel }) {
 		this.logger = logger;
 		this.transactionPool = transactionPool;
 		this.config = broadcasts;
 		this.channel = channel;
-		this.storage = storage;
 
 		this.queue = [];
-
-		// Broadcast routes
-		this.routes = [
-			{
-				path: 'postTransactionsAnnouncement',
-				collection: 'transactions',
-				object: 'transaction',
-			},
-			{
-				path: 'postSignatures',
-				collection: 'signatures',
-				object: 'signature',
-			},
-		];
+		this.transactionIdQueue = [];
+		this.signatureObjectQueue = [];
 
 		if (this.config.active) {
-			jobsQueue.register(
-				'broadcasterReleaseQueue',
-				async () => this.releaseQueue(),
-				this.config.broadcastInterval,
-			);
+			setInterval(async () => {
```

Comment:
Just for curiosity, why not using jobsQueue anymore?

Reply:
I think jonsQueue doesn't make sense in a way that it's every 5 sec, so it will never be stuck in the execution. So in this case i thought `setInterval` is enough